.github/ISSUE_TEMPLATE/TASK.yaml

```
@@ -1,11 +1,75 @@
----
-name: 🏃🏻‍♂️ Task
-description: A general task that does not fit into other categories.
-title: "[task]: "
-type: Task
+name: "Task"
+description: "Work item that goes through refinement before development"
+title: "[Task] <short title>"
+labels: ["task"]
```

We should assign it the Task type, and not a label. We should not use labels for issue types any more, as we have the types for that now. I am not sure GitHub has documented this yet, but it is possible with the below change.

```suggestion
type: Task
```

With that, I also do not think a default label is necessary.